Dockerfile

```diff
@@ -28,10 +27,7 @@ RUN opam init \
         --shell-setup \
         --yes \
     && opam switch create ${HEMLOCK_BOOTSTRAP_OCAML_VERSION} \
-    && opam install -y \
-        ocaml-lsp-server \
-        ocp-indent \
-        utop \
+    && opam install -y ocp-indent \
```

Comment:
[Here](https://github.com/cevans87/dotfiles/blob/master/opam/hemlock/general#L1-L2)'s an example of how I moved this into my dotfiles.